xapian-letor/feature/feature.cc

```diff
@@ -36,10 +39,17 @@ void
 Feature::operator=(const Feature & o)
 {
     LOGCALL_VOID(API, "Feature::operator=", o);
-    internal = o.internal;
+    feature_db = o.feature_db;
+    feature_query = o.feature_query;
+    feature_doc = o.feature_doc;
+    termfreq = o.termfreq;
+    inverse_doc_freq = o.inverse_doc_freq;
+    doc_length = o.doc_length;
+    collection_length = o.collection_length;
+    collection_termfreq = o.collection_termfreq;
 }
```

Comment:
Do `Feature` objects need to be copyable and/or assignable?  If not, I'd be tempted to not allow copying or assignment - they're both going to be costly operations and we tell users that API classes are cheap to copy and assign.